docs-wiki/Governing-board.md

```diff
@@ -0,0 +1,87 @@
+  - Defining board roles and responsibilities.
+  - Managing the GitHub community (e.g., triaging issues, answering discussions, reviewing PRs).
+  - Establishing an official support process.
+  - Managing and supporting contributors and engaging with new contributions.
+  - Identifying and prioritizing initiatives.
+- **Performance**: Driving and quantifying value for consumers and contributors, including:
+  - External communications about releases (to drive adoption/engagement).
+  - Internal communications about customer impact (to justify contributors' time).
+  - Telemetry to measure satisfaction, engagement, adoption, retention, etc.
+- **Compliance**: Ensuring we're in compliance with Microsoft policies and procedures.
+
+<br>
+
+## 🤔 Making decisions
+
+The board follows a [consensus decision-making process](https://en.wikipedia.org/wiki/Consensus_decision-making#Objectives), which means:
+
+1. **Collaborative**: Members work together to produce a shared proposal that meets the concerns of all group members as much as possible through constructive contributions.
+2. **Cooperative**: Members strive to reach the best possible decision for the contributors, contributions, and consumers alike, rather than competing for personal preferences.
+3. **Egalitarian**: All members have an equal opportunity to present and amend proposals.
+4. **Inclusive**: As many board and council members as possible should be involved in the decision-making process.
+5. **Participatory**: Members actively solicit the input and participation of all advisory council members and any additional, underrepresented stakeholders.
+
+Decisions are generally made via "lazy approval", where consent is assumed unless valid objections are raised within 3-7 days. Decisions are approved when the board attains a supermajority (two-thirds) consensus and no objections. Any board or council member can request a 1-week extension of the deadline or to request an explicit vote across all board and council members.
+
+Objections must have a clear justification, impact statement, and suggestions to make the proposal acceptable. If an objection is not actionable, it can be dismissed with a supermajority vote.
+
+In general, board decisions should be open and transparent for any contributor or consumer upon request and documented in the repository. Any FinOps toolkit contributor or consumer can make an objection. The advisory council is responsible for ensuring concerns and objections from others are raised, but only council members' objections are binding.
+
+If consensus cannot be reached within 14 days and the board believes the issue is deadlocked and unable to proceed, the board chair may decide the final resolution.
+
+<br>
+
+## 🧑‍🤝‍🧑 Composition
+
+The governing board consists of the following representative seats:
+
+- Community
+- Consulting
+- Engineering
+- Marketing
+- Product
+- FinOps Certified Professional
+- Internal customer (up to 2)
+- External customer (up to 3)
```

Comment:
Should we be more explicit about the types of internal/external customers we want on the board? For instance, do we want to require they be a FinOps practitioner or should we have separate seats for each FinOps persona? We can also figure this out later since we don't have nominees fighting for a seat but it might be good to have an early opinion before we get to that point.